src/clients/api/queries/getVoteDelegates.ts

```diff
@@ -0,0 +1,21 @@
+import { XvsVault } from 'types/contracts';
+
+export interface IGetVoteDelegatesInput {
+  xvsVaultContract: XvsVault;
+  accountAddress: string;
+}
+
+export type GetVoteDelegatesOutput = string;
+
+/**
+ *
+ * @param address string (valid Ethereum address)
+ * @returns Delegated address, if no delegation returns null address
```

Comment:
We could return undefined if the contract returns the null address, this way the front end doesn't need to important the null address from anywhere and we avoid risking that a dev who doesn't have this context simply displays the null address thinking it's valid.